spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/BoxControllerHelperService.java

```diff
@@ -621,62 +617,34 @@ public JSONObject getBoxBarcode(HttpSession session, JSONObject json) {
    * @return JSON message indicating success or error
    */
   public JSONObject printBoxBarcodes(HttpSession session, JSONObject json) {
-    try {
-      User user = authorizationManager.getCurrentUser();
+    return ControllerHelperServiceUtils.printBarcodes(securityManager, printManager, json, new BarcodePrintAssister<Box>() {
```

Comment:
If I do that, I can't reuse them in the `ProjectControllerHelperService`. There's minimal cost of instantiating it if that's your concern.